File: Easy/JavaScript/1925. Count Square Sum Triples.js
var countTriples = function(n) {
    let count = 0;

    for(let i = 1; i < n - 1; i++){
        for(let j = i + 1; j < n; j++){
            for(let k = j + 1; k < n + 1; k++){
                if((i * i) + (j * j) == (k * k))
                    count += 2;
            }
        }
    }

    return count;
};

File: Easy/Java/1925. Count Square Sum Triples.java
class Solution {
    public int countTriples(int n) {
        int count = 0;

        for(int i = 1; i < n - 1; i++){
            for(int j = i + 1; j < n; j++){
                for(int k = j + 1; k < n + 1; k++){
                    if((i * i) + (j * j) == (k * k))
                        count += 2;
                }
            }
        }

        return count;
    }
}
